### DISCUSSION TITLE
has_path()

### DISCUSSION THREAD

**Discussion by asthapoudel** [networkx/networkx#6433] [General]
What algorithm is specifically used by has_path() funnction. I wanted to know about the details and methods it uses.

**Comment by rossbar** [comment]
The beauty of open-source is that you can see how things are implemented for yourself!: https://github.com/networkx/networkx/blob/0f91550007fd3a95261d858b1a6a623ef8bda38a/networkx/algorithms/shortest_paths/generic.py#L20
